udp-native/src/main/java/org/ballerinalang/stdlib/udp/UdpListener.java

```diff
@@ -81,9 +81,8 @@ public static void send(UdpService udpService, DatagramPacket datagram, Channel
     // only invoke if the listener is a connected listener
     private void connect(SocketAddress remoteAddress, SocketAddress localAddress, Future callback)
             throws InterruptedException {
-        ChannelFuture channelFuture = listenerBootstrap.connect(remoteAddress).sync();
-        channel = channelFuture.channel();
-        channel = listenerBootstrap.bind(localAddress).sync().channel();
+        ChannelFuture channelFuture = listenerBootstrap.connect(remoteAddress, localAddress).sync();
+        channel = channelFuture.sync().channel();
```

Comment:
Can't we remove invoking sync() here as I mentioned in the previous comment ?

Reply:
can , updated ```
 listenerBootstrap.connect(remoteAddress, localAddress).sync().addListener((ChannelFutureListener) future -> {
            channel = future.channel();
            if (future.isSuccess()) {
                callback.complete(null);
            } else {
                callback.complete(Utils.createSocketError("Can't connect to remote host."));
            }
        });
```